Makefile

```diff
@@ -23,13 +23,23 @@ pre-commit:
 	python3 -m flake8
 	python3 -m mypy sematic
 	python3 -m black sematic --check
```

Comment:
Not removing `black` too?

Reply:
I will; I'm going to get things working in parallel before removing the old tooling